company/okrs/2020_q1.md

```diff
@@ -0,0 +1,106 @@
+   1. [$N][N] net new ARR
+   1. Sales leadership owns revenue targets and hiring plan
+   1. Sales, engineering, and product have a working process for prioritization and delivery
+1. **CEO: Aspirational, efficient all-remote team**
+   1. Roles and responsibilities are documented and maintained in handbook
+   1. Team members rely on the handbook for cross-functional info and processes
+   1. The handbook is fast to browse and search
+
+## Sales and marketing
+
+1. **Sales (Closing Deals): Achieve [$N][N] net new ARR.**
+   1. Have each AE close 3 Tier 1 deals (>=$100K ARR) and 10 non-Tier 1 deals, 2 of which are from their own outbound outreach efforts.
+   1. Get 70% of companies on a free trial to convert to paid customers.
+1. **Sales (Pipeline Optimization): Scale our sales pipeline to 50+ deals per AE, of which 60% of deals are engaged and actively evaluating Sourcegraph.**
+   1. Round robin new instances across team to collectively achieve a 15%+ response rate from Prospects who set up a new Sourcegraph instance.
+
+1. **VP Business: Build a Customer Success organization that supports and helps us exceed our ARR goals.**
+   1. Generate at least [$M][M] in net new ARR from existing accounts.
+   1. Hire a Head of Customer Success who can own this function.
+   1. Reach monthly NPS of 60 at all Tier 1 customers.
+
+1. **VP Business: Make Sourcegraph a lean organization that uses budgets to guide decision making.**
+
+1. **VP Marketing: Deliver sufficient MQLs to sales to reach [$N][N] net new ARR goal.**
+1. **VP Marketing: Create new category of developer tool.**
+1. **VP Marketing: Become the dominant provider of source code search.**
+
+## Product
+
+1. **Product: Create a customer driven product process**
+   1. All product engineering work has a delivery plan (where "delivery plan" links to a new handbook page that defines it, based on [RFC 97](https://docs.google.com/document/d/1cZ7JIVuRWrF2MxwDdH36SC7zOyT2qJf9AMUd9Wc9_aY/edit#heading=h.dkow31it1qt1)).
+   1. Handbook describes the lifecycle of a feature and who is responsible for each part of the lifecycle.
+   1. Handbook describes product planning processes and API between product and engineering.
+1. **Product: Build the product and design team to support engineering and ARR growth**
+   1. Hire +1 UX Designer.
+   1. Hire +1 Product Manager.
+
+## Engineering
+
+### VP Eng
+
+1. **Build a world class engineering team.**
+   1. Grow team to plan (+1 code intel, +2 distribution, +2 web, +1 core services).
+   1. Transition an existing teammate to be (or hire) an Engineering Manager of the distribution team.
+   1. Document how we do performance reviews and determine compensation for teammates in engineering roles.
+1. **Create a data driven planning process.**
+   1. Each team commits to a written work plan for each release cycle.
+   1. Each team completes >80% of the work that they committed to at the beginning of the release cycle each iteration.
+
+### Distribution
```

Comment:
@nicksnyder updated following our conversation in Distribution sync today

Reply:
Thanks! I left one comment below.